cp-ksqldb-cli/Dockerfile.ubi8

```diff
@@ -52,9 +52,25 @@ name=Confluent repository \n\
 baseurl=${CONFLUENT_PACKAGES_REPO}/ \n\
 gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
-enabled=1 " > /etc/yum.repos.d/confluent.repo \
-    && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+enabled=1 " > /etc/yum.repos.d/confluent.repo
+
+RUN echo "===> Installing Confluent Hub client ${CONFLUENT_VERSION}..." \
+    && yum install -y confluent-hub-client-${CONFLUENT_VERSION}
+
+RUN echo "===> HACK to make cc-role-definitions available in docker image..." \
```

Comment:
NIT: If we want to run this conditionally via an external arg so that it can be excluded/included easily.

RUN if [[ -z "$arg" ]] ; then echo Argument not provided ; else echo Argument is $arg ; fi